ClassHierarchyAndCollections/TestClassHierarchyAndCollections/ClassTest.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ClassHierarchyAndCollections;
+
+namespace TestClassHierarchyAndCollections
+{
+    /// <summary>
+    /// Defines the test for the <see cref="ClassHierarchyAndCollections"/> class.
+    /// </summary>
+    [TestClass]
+    public class ClassTest
+    {
+        [TestMethod]
+        public void Contact_Is_Contact()
+        {
+            Contact bob = new Contact("Bob", "123 Somewhere Lane", "123-456-7890");
+            Assert.IsInstanceOfType(bob, typeof(Contact));
+        }
+        public void Organization_Is_Organization()
```

Comment:
Only your first test method is decorated with the [TestMethod] attribute meaning that only one can be run as a test. I added [TestMethod] to each of the other methods to ensure they pass but 9 tests are currently failing. You also have not tested that you can read from and write to your public properties. (-2.0)